code/datums/mutations/sight.dm

```diff
@@ -42,7 +41,6 @@
 	difficulty = 18
 	text_gain_indication = "<span class='notice'>You can see the heat rising off of your skin...</span>"
 	time_coeff = 2
-	instability = 40
```

Comment:
Leave instability the same as they're still the same effectiveness use-wise if someome *somehow* gains these (and if admins don't want it to have it as exception during bussing they can just manually adjust it)